### ISSUE TITLE
uninstallb could remove associated conda envs

### ISSUE THREAD

**Issue by azazellochg** [scipion-em/scipion-app#173]
to save disc space. At the moment it only deletes folders inside software/em, which are usually empty for many conda-based binaries
I'd imagine we could parse the env name from the activation command and then delete it (with user confirmation?)

**Comment by pconesa** [comment]
I think we need to provide some kind of "uninstallCommands"

**Comment by azazellochg** [comment]
i dont want to type "uninstallCommands", that's like 20 letters! :)

**Comment by pconesa** [comment]
I don't see other option.

**Comment by pconesa** [comment]
yes, unless we parse the commands and look for a conda create something which may not be robust enough?

**Comment by azazellochg** [comment]
we could check for ...ENV_ACTIVATION var, but it is also not very reliable. Maybe we could add another (boolean?) parameter to PluginInfo? I don't know if this class is used to store plugin properties..
